apps/src/music/types.ts

```diff
@@ -36,6 +36,7 @@ export interface MusicLevelData extends ProjectLevelData {
   aiCodeGenerateExtraPrompt?: string;
   // Dance move to show when playing music.
   danceMove?: string;
+  guideMode?: string;
```

Comment:
Do we have discrete options for this we can use?

Reply:
Yes, done.